tests/tox.ini

```diff
@@ -25,7 +25,7 @@ passenv =
 commands =
   pip install ../client
   pip check
-  python -m pytest -v --order-dependencies
+  python -m pytest -v -s --order-dependencies
```

Comment:
The `-s` is needed to print the logs. Pytest, by default, capture stdout. With this flag, the capture is disabled:
- More info: https://docs.pytest.org/en/stable/how-to/capture-stdout-stderr.html